depends/packages/native_ccache.mk

```diff
@@ -1,8 +1,8 @@
 package=native_ccache
-$(package)_version=3.4.3
-$(package)_download_path=https://samba.org/ftp/ccache
-$(package)_file_name=ccache-$($(package)_version).tar.bz2
-$(package)_sha256_hash=38decec3eed6524a002ffae52b25781843553c28e2d840e7e55e6831c40264d2
+$(package)_version=3.7.12
+$(package)_download_path=https://github.com/ccache/ccache/releases/download/v3.7.12
+$(package)_file_name=ccache-3.7.12.tar.gz
```

Comment:
$(package)_download_path=https://github.com/ccache/ccache/releases/download/v$($(package)_version)
$(package)_file_name=ccache-$($(package)_version).tar.gz

Wasn't sure if this was changed out intentionally or just overlooked after testing was complete.